eng/common/instructions/azsdk-tools/typespec-to-sdk.instructions.md

```diff
@@ -88,19 +88,29 @@ Your goal is to guide user through the process of generating SDKs from TypeSpec
 **Actions**:
 1. Run `GetSDKPullRequestDetails` to fetch generated SDK PR info.
 
-## Step 9: Create release plan
+## Step 9: Validate Label and Codeowners
+**Goal**: Validate the label and all codeowners for a service. Create new label and codeowner entry if none exist.
+**Actions**:
+1. To validate a service label refer to #file:./validate-service-label.prompt.md
+2. After service label is validated or created refer to #file:./validate-codeowners.prompt.md
+3. Handle post-validation actions based on results:
+   - **If both label and codeowners were already valid**: Prompt user "Your service label and codeowners are already properly configured. Would you like to modify the existing codeowners entry for your service?"
+   - **If new label or codeowner entries were created**: Display details of all created PRs, then prompt user "The following PRs have been created for your service configuration: [list PRs]. Would you like to make any additional modifications to these entries?"
```

Comment:
why is it necessary to print the PRs ? In theory, step 8 displayed them. 
If we need it, should we ask for it too on line 97?